README.md

```diff
@@ -1 +1,28 @@
-# Conduit_Container
\ No newline at end of file
+# Conduit_Container
+
+A Docker Compose container that builds two containers, a Conduit backend and a Conduit frontend.
+
+## Requirements
+
+- Docker Compose
+
+## Quickstart
+
+1. clone this repository
+
+    ```bash
+    git clone git@github.com:JohnTwiiX/Conduit_Container.git
+    ```
+
+1. start the conatiner
```

Comment:
Would rather say: "Navigate to repo" followed by the `cd` command (maybe even inline -> same line). Then in the next line you can do something like start the container with "docker compose up -d", with the info that `-d` stands for detached mode and a litte explanation what this exactly means